core/src/main/java/org/mapstruct/Mapper.java

```diff
@@ -357,4 +357,16 @@ NullValuePropertyMappingStrategy nullValuePropertyMappingStrategy() default
      * @since 1.4
      */
     Class<? extends Exception> unexpectedValueMappingException() default IllegalArgumentException.class;
+
+    /**
+     * Flag indicating whether the creation of a time stamp in the {@code @Generated} annotation should be suppressed.
+     * i.e. not be created.
+     *
+     * The method overrides the flag set in a central configuration set by {@link #config()}
```

Comment:
mention link to `MapperConfig` annotation..?

Reply:
Not following. This is identical to the other places. Check `componentModel`

Reply:
ok.. leave as is.